lib/TokenManager.js

```diff
@@ -23,20 +80,27 @@ function add(storage, key, token) {
   }
   tokenStorage[key] = token;
   storage.setTokenStorage(tokenStorage);
+  if (tokenMgmtRef.autoRefresh) {
+    setRefreshTimeout(sdk, tokenMgmtRef, storage, key, token);
```

Comment:
Also here - it seems like we might want another method that schedules emitting that event if we're not autoRefreshing.